lib/active_support/cache/memcached_store.rb

```diff
@@ -44,6 +44,8 @@ def initialize(*addresses)
           @data = Memcached::Rails.new(*(addresses + [mem_cache_options]))
         end
 
+        @deal_with_exceptions = options.fetch(:deal_with_exceptions, true)
```

Comment:
I would suggest a name like `swallow_exceptions` or `treat_exception_as_miss`—I'd find that a little bit more descriptive when using this